tests/test_parallel_post_fit.py

```diff
@@ -61,13 +62,16 @@ def test_predict_meta_override():
     base.fit(pd.DataFrame(X), y)
 
     dd_X = dd.from_pandas(X, npartitions=2)
-    dd_X._meta = pd.DataFrame({"c_0": [5]})
 
-    # Failure when not proving predict_meta
-    # because of value dependent model
-    wrap = ParallelPostFit(base)
-    with pytest.raises(ValueError):
-        wrap.predict(dd_X)
+    if not DASK_EXPR_ENABLED:
+        # dask-expr cannot set _meta
+        dd_X._meta = pd.DataFrame({"c_0": [5]})
+
+        # Failure when not proving predict_meta
+        # because of value dependent model
+        wrap = ParallelPostFit(base)
+        with pytest.raises(ValueError):
+            wrap.predict(dd_X)
```

Comment:
@phofl I kinda assume we can safely ignore this test for dask-expr? Fails b/c we cannot set `_meta`.

Reply:
Hm this is not great since it seems like the behaviour actually depends on meta?

Reply:
Is there a 'good' way to modify meta? Otherwise I can try mocking it here.

Reply:
lets chat about this in the standup

Reply:
okay lets move ahead then